sql/scan.go

```diff
@@ -403,7 +405,7 @@ func (n *scanNode) computeOrdering(columnIDs []ColumnID) []int {
 		if !found {
 			// Exit if the column ID in the index is not one of the output
```

Comment:
comment needs an update; perhaps the comment you posted in response to my question about `p == 0`